pkg/ccl/backupccl/backup_test.go

```diff
@@ -1502,13 +1502,23 @@ func TestBackupRestoreResume(t *testing.T) {
 
 	params := base.TestClusterArgs{ServerArgs: base.TestServerArgs{
 		Knobs: base.TestingKnobs{JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals()},
+
+		// Cannot run this test within tenant because it panics
+		// https://github.com/cockroachdb/cockroach/issues/88527
+		DisableDefaultTestTenant: true,
 	}}
 
 	const numAccounts = 1000
 	tc, outerDB, dir, cleanupFn := backupRestoreTestSetupWithParams(t, multiNode, numAccounts, InitManualReplication, params)
 	defer cleanupFn()
-
-	backupTableDesc := desctestutils.TestingGetPublicTableDescriptor(tc.Servers[0].DB(), keys.SystemSQLCodec, "data", "bank")
+	codec := keys.SystemSQLCodec
+	testTenants := tc.Servers[0].TestTenants()
+	clusterID := tc.Servers[0].RPCContext().LogicalClusterID.Get()
+	if len(testTenants) > 0 {
+		codec = keys.MakeSQLCodec(testTenants[0].RPCContext().TenantID)
+		clusterID = testTenants[0].RPCContext().LogicalClusterID.Get()
+	}
```

Comment:
Locally, I've added:

```
func (tc *TestCluster) TenantOrServer(idx int) serverutils.TestTenantInterface {
	s := tc.Server(idx)
	if tc.StartedDefaultTestTenant() {
		return s.TestTenants()[0]
	}
	return s
}
```

which might be a nice addition to make this a bit cleaner throughout the tests.